src/app/gene-plot/gene-plot.component.css

```diff
@@ -38,6 +38,10 @@ label {
   overflow: hidden;
 }
 
+.header-item:nth-child(3) span{
```

Comment:
Unsafe css, this might apply to a different element than expected if the order is changed. A better solution would be to add an id or class to the Summary alleles/Family variants elements and use a more specific selector